ZotBinsAPI.py

```diff
@@ -19,6 +21,11 @@
 
 @app.route('/')
 def index():
+    """
+    Test that you setup the API correctly with this endpoint.
+
+    Call http://localhost:[ENTER_PORT_HERE]/
```

Comment:
instead of `localhost`, should it be the zotbins pythonanywhere url?

Reply:
Could be either. I used `localhost` here because I intended on the developer testing their local setup first.

You have a point though-- it would probably be a good idea to include the pythonanywhere URL so that they can check either their local setup or if they are actually connecting to the server.